experiment_prototype/experiment_prototype.py

```diff
@@ -262,11 +264,12 @@ def __init__(self, cpid):
         self.__new_slice_id = 0
 
         # Centre frequencies can be specified in your experiment class using the setter. TODO: make modifiable (with warning that it takes time. Get time estimate for this.
-        self.__txctrfreq = 13000  # in kHz.
-        self.__rxctrfreq = 13000  # in kHz.
+        self.txctrfreq = 12000  # in kHz.
```

Comment:
This will call the setter but after a google, apparently we can also define the attribute here in __init__ using this:

`self.__txctrfreq = self.txctrfreq = 12000`
`self.__rxctrfreq = self.rxctrfreq = 12000`

Reply:
Yeah we could also do it that way. it does look a bit more clear that way.